app/com/m3/octoparts/model/config/HttpPartConfig.scala

```diff
@@ -71,20 +79,25 @@ object HttpPartConfig {
     JsonHttpPartConfig(
       partId = config.partId,
       owner = config.owner,
-      uriToInterpolate = config.uriToInterpolate,
       description = config.description,
+      uriToInterpolate = config.uriToInterpolate,
       method = config.method,
       hystrixConfig = HystrixConfig.toJsonModel(config.hystrixConfig.get),
       additionalValidStatuses = config.additionalValidStatuses,
+      httpPoolSize = config.httpPoolSize,
+      httpConnectionTimeout = config.httpConnectionTimeout,
+      httpSocketTimeout = config.httpSocketTimeout,
+      httpDefaultEncoding = config.httpDefaultEncoding,
+      httpProxy = config.httpProxy,
       parameters = config.parameters.map(PartParam.toJsonModel),
       deprecatedInFavourOf = config.deprecatedInFavourOf,
       cacheGroups = config.cacheGroups.map(CacheGroup.toJsonModel),
       cacheTtl = config.cacheTtl,
-      alertMailsEnabled = config.alertMailsEnabled,
-      alertAbsoluteThreshold = config.alertAbsoluteThreshold,
-      alertPercentThreshold = config.alertPercentThreshold,
-      alertInterval = config.alertInterval,
-      alertMailRecipients = config.alertMailRecipients,
+      alertMailSettings = AlertMailSettings(alertMailsEnabled = config.alertMailsEnabled,
```

Comment:
Nice job introducing `AlertMailSettings`. It would be nice if we could do this too in the constructor and for HttpClientConfig